docs/VMR-Product-Versioning.md

```diff
@@ -0,0 +1,164 @@
+        RC1Stable --> RC1Release["Release (RC 1)"]
+    end
+
+    subgraph RCBranchRC2["RC Branch 2"]
+        RC2Stable["Stable Builds (RC 2)"]
+        RC2Stable --> RC2Release["Release (RC 2)"]
+    end
+
+    subgraph ServicingBranch[".NET N.0.0 Servicing Branch"]
+        RC1Development[RC 1 development]
+        RC1Development -->|Rebrand to RC 2| RC2Development["RC 2 Development"]
+        RC2Development -->|Rebrand to GA| GADevelopment["GA Development"]
+        GADevelopment --> GARelease["Release GA"]
+        GARelease -->|Rebrand to N.0.1| ServicingRelease1Development["Servicing release 1 development"]
+        ServicingRelease1Development --> ServicingRelease1["Release N.0.1"]
+        ServicingRelease1 -->|Rebrand to N.0.2| ServicingRelease2Development["Servicing release 2 development"]
+        ServicingRelease2Development --> ServicingRelease2["Release N.0.2"]
+        ServicingRelease2 --> DotDotDotServicing[...]
+    end
+
+    %% Connections between branches
+    P1 -->|Branch for P1| PreviewBranchP1
+    P2 -->|Branch for P2| PreviewBranchP2
+    P3 -->|Branch for P3| PreviewBranchP3
+    P4 -->|Branch for P4| PreviewBranchP4
+    P5 -->|Branch for P5| PreviewBranchP5
+    P6 -->|Branch for P6| PreviewBranchP6
+    P7 -->|Branch for P7| PreviewBranchP7
+    P7 -->|Branch and rebrand to RC 1| ServicingBranch
+    RC1Development -->|Branch for RC 1| RCBranchRC1
+    RC2Development -->|Branch for RC 2| RCBranchRC2
+```
+
+## Open questions
+
+There are a few open questions that may require some experience before settling on a final answer:
+
+### How should branding be updated? Centrally in VMR or individual repos?
+
+Branding has been historically updated in individual repos. With the VMR, we could now do branding centrally, checking in an update in a single commit.
+
+**Proposal: Update branding centrally in the VMR and backflow to repos**
+
+### Should stable branding be 'checked-in'?
+
+In the past, we would activate stable branding for a repository by changing its Versions.props. All future builds of that branch would be stable. The VMR now has options to queue a build with stable branding. Does stable branding need to be checked in any longer? Or can it just be activated on-demand with a switch?
+
+**Proposal: Check-in stable branding in preview and servicing branches. This ensures that rolling CI is producing the stable bits required for release at all times.**
+
+### If checked in, how should stable branding be activated?
+
+There are options in the VMR build pipeline to activate stable branding (preview or rtm release branding) via pipeline parameter. In addition, the stable switches could also be activated in the VMR repo-project Directory.Build.props, or individual repo Versions.props.
+
+**Proposal: Use YAML pre-processor directives to change defaults for stable branding to produce 'prerelease stable' in release/preview.N or release/rc.N, and 'rtm stable' in release/N, and non-stable everywhere else.**
```

Comment:
When checking out the release/10.0.1xx branch in the VMR let's say in November, I won't need to pass in an additional arg to produce stable branded bits or would I? And would that only get enabled when doing a local OfficialBuildId build?

Reply:
You would. The CI would pass -branding rtm by default.